concrete-commons/src/parameters.rs

```diff
@@ -102,6 +102,10 @@ pub struct PolynomialCount(pub usize);
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
 pub struct MonomialDegree(pub usize);
```

Comment:
Shouldn't we remove MonomialDegree?

Reply:
It is still needed in the core backend, so I did not removed it just yet...

Reply:
Hum, shouldn't we mark it as deprecated just in case modifying the backend comes in some time?

Reply:
Excellent idea ! I added the deprecation note